packages/contracts/slither.config.json

```diff
@@ -1,9 +1,10 @@
 {
-  "exclude_informational": false,
-  "exclude_low": false,
+  "exclude_informational": true,
+  "exclude_low": true,
+  "detectors_to_exclude": "naming-conventions,different-pragma-directives-are-used,external-function,assembly,incorrect-equality,incorrect-versions-of-solidity,unused-return",
   "exclude_medium": false,
   "exclude_high": false,
   "disable_color": false,
-  "filter_paths": "(lib/|test/)",
+  "filter_paths": "lib|test|src/dollar/mocks|src/ubiquistick",
```

Comment:
Why is slither skipping ubiquistick? I also remember seeing this on another PR some time ago.

Reply:
Ideally it should be included in slither checks but I figured out it requires a lot of changes. Fixing slither issue would take lots of time rather than re-implementing them properly. so I made it excluded. In the near future, we should include again.